tests.py

```diff
@@ -279,6 +280,36 @@ def test_warm_start(self):
         np.testing.assert_allclose(y, y_p, atol=1e-7)
         np.testing.assert_allclose(s, s_p, atol=1e-7)
 
+    def test_threading(self):
+        m = 200
+        n = 100
+        As, bs, cs, cone_dicts = [], [], [], []
+        results = []
+
+        serial_time = 0.0
+        for _ in range(20):
+            A, b, c, cone_dims = utils.least_squares_eq_scs_data(m, n)
+            As += [A]
+            bs += [b]
+            cs += [c]
+            cone_dicts += [cone_dims]
+            tic = time.time()
+            results.append(cone_prog.solve_and_derivative(A, b, c, cone_dims))
+            toc = time.time()
+            serial_time += toc - tic
+
+        tic = time.time()
+        results_thread = cone_prog.solve_and_derivative_batch(As, bs, cs, cone_dicts)
+        toc = time.time()
+        parallel_time = toc - tic
+
+        print(parallel_time, serial_time)
```

Comment:
nit, if you want to print these stats, include some text (e.g., "parallel time {:f}, serial time {:f}".format(parallel_time, serial_time))